js/bootstrap-datepicker.js

```diff
@@ -132,6 +132,9 @@
 		this.setEndDate(this._o.endDate);
 		this.setDaysOfWeekDisabled(this.o.daysOfWeekDisabled);
 		this.setDatesDisabled(this.o.datesDisabled);
+		if (options.startDate){
```

Comment:
I think you should replace the chech here with `o.startDate !== -Infinity` as the default value for the startDate is infinity